.github/workflows/ci_build.yml

```diff
@@ -43,6 +43,18 @@ jobs:
 
           cd ..
           python3 build/compose.py --use-all --obsolete-fillers gfx/UltimateCataclysm
+
+          for file in gfx/UltimateCataclysm/*.png; do
+            convert "$file" "$file.tmp"
```

Comment:
```
convert -type optimize -define png:compression-strategy=0 -define png:compression-level=9 -define png:exclude-chunk=EXIF,iCCP,iTXt,sRGB,tEXt,zCCP,zTXt,date "$file" "$file.tmp"
```
Honestly, imo, `-type optimize` isn't necessary, but it's worth having.

Reply:
Applying but don't have time to check results right now, thank you!

Reply:
Even better, please convert your comment to change suggestion (ctrl+g, `±` button in the edit interface) so that you will be properly added as a `Co-authored-by` without me needing to spending time on it :)

Reply:
![зображення](https://user-images.githubusercontent.com/1585431/107940005-4051ab80-6f90-11eb-99c3-4e6b973a0b76.png)
Looks like the images are already pretty optimized but these options still were able to save some bytes :+1: